Xamarin.Forms.Platform.UAP/BoxViewRenderer.cs

```diff
@@ -23,7 +23,8 @@ protected override void OnElementChanged(ElementChangedEventArgs<BoxView> e)
 					};
 
 					rect.SetBinding(WShape.FillProperty, new Windows.UI.Xaml.Data.Binding { Converter = new ColorConverter(), Path = new PropertyPath("Color") });
-	
+					rect.SetBinding(WShape.FillProperty, new Windows.UI.Xaml.Data.Binding { Converter = new BrushConverter(), ConverterParameter = e.NewElement.Color, Path = new PropertyPath("Background") });
```

Comment:
Is this code correct? It seems like this will overwrite the previous binding?

Reply:
Not at all, with the second binding the first one is no longer neccesary.